eth/db/header.py

```diff
@@ -198,13 +204,24 @@ def _set_as_canonical_chain_head(self, block_hash: Hash32) -> Tuple[BlockHeader,
             )
 
         new_canonical_headers = tuple(reversed(self._find_new_ancestors(header)))
+        old_headers = []
+
+        for h in new_canonical_headers:
+            try:
+                old_hash = self.get_canonical_block_hash(h.block_number)
+            except HeaderNotFound:
+                # no old block, and no more possible
+                break
+            else:
+                old_header = self.get_block_header_by_hash(old_hash)
```

Comment:
Can we update and use the same naming convention `orphaned_` in this function body?